langserver/langserver_test.go

```diff
@@ -1215,16 +1219,16 @@ func lspTests(t testing.TB, ctx context.Context, fs *AtomicFS, c *jsonrpc2.Conn,
 		wantGodefHover = cases.wantHover
 	}
 
-	if len(wantGodefDefinition) > 0 || (len(wantGodefHover) > 0 && fs != nil) || len(cases.wantCompletion) > 0 {
-		UseBinaryPkgCache = true
+	if len(wantGodefDefinition) > 0 || (len(wantGodefHover) > 0 && h != nil) || len(cases.wantCompletion) > 0 {
+		h.config.UseBinaryPkgCache = true
```

Comment:
There may be a good case for refactoring this more, to avoid mutating the config mid-test, but I felt this was out of scope of this PR.

Reply:
Given this was mutating a global variable and never returning it to the old value, this is actually better. Maybe we can just always have `UseBinaryPkgCache` set to true?

Reply:
It gets reverted to `false` at [langserver_test.go:1277](https://github.com/pbitty/go-langserver/blob/68714b56f864bd53c7e80faa132e9d30c0c173e2/langserver/langserver_test.go#L1277).

Reply:
aah I missed that. This this seems good to me.